lib/rucio/common/schema/__init__.py

```diff
@@ -138,12 +140,18 @@ def load_schema_for_vo(vo: str) -> None:
 
 
 def validate_schema(name: str, obj: Any, vo: str = 'def') -> None:
-    if vo not in schema_modules:
-        load_schema_for_vo(vo)
-    if not hasattr(schema_modules[vo], 'validate_schema'):
-        _get_generic_schema_module().validate_schema(name, obj)
-        return
-    schema_modules[vo].validate_schema(name, obj)
+    if obj:
```

Comment:
Is this `if obj` intentional in it's depth? I.e. it doesn't validate empty dicts for required properties or empty lists that have a jsonschema that might require at least 1 item in the list?